enclave-runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "enclave-runner"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
This should be a minor version bump due to tokio 0.2 stuff that landed recently.

Reply:
That change doesn't change the public API though, right?

Reply:
They change the return type of things in `UsercallExtension`